app/Resources/views/articleInfo/result.html.twig

```diff
@@ -92,7 +107,7 @@
                         </td>
                         <td>{{ ai.numEditors|num_format }}</td>
                     </tr>
-                    {% if ai.assessments and not (page.isMainPage) %}
+                    {% if ai.assessments and not (page.isMainPage) and not (page.hasDateRange) %}
```

Comment:
`ai.hasDateRange` (not `page`), and again I'd put this check before `ai.assessments`